documentation/RELEASE_NOTES.md

```diff
@@ -0,0 +1,1677 @@
+# EPICS Base Release 3.15.7
```

Comment:
I'd like git to see this file as a rename and subsequent edit of the RELEASE_NOTES.html file; git detects renames, and they make other people's work merge better if it knows that history. I'd like to see this change done as two separate commits:

1. `git mv documentation/RELEASE_NOTES.html documentation/RELEASE_NOTES.md`
`git commit -m "Rename Release Notes to .md"`
2. Replace documentation/RELEASE_NOTES.md content with your converted version, then:
`git add documentation/RELEASE_NOTES.md`
`git commit`

You could do the same for the README file but it isn't necessary since it very rarely gets changed. It would be nice for it that to be in a separate commit from the Release Notes conversion though.